src/Collection/CollectionInterface.php

```diff
@@ -43,7 +43,7 @@
      * in this collection
      * @return \Cake\Collection\CollectionInterface
```

Comment:
should be $this then right?

Reply:
Collection methods don't return $this as they are an immutable style class. They return a new clone like the request/response.